modules/web/src/app/cluster/list/cluster/template.html

```diff
@@ -14,11 +14,20 @@
 limitations under the License.
 -->
 
-<mat-card class="km-card">
+<mat-card fxLayout="column"
+class="km-card">
   <mat-card-header>
     <mat-card-title>Clusters</mat-card-title>
   </mat-card-header>
-  <div fxLayout
+  <div *ngIf="!!projectClusterListErrorMessage"
+  fxLayoutAlign=" center"
+  class="error-message">
+<i class="km-icon-warning"></i>
+<div>{{projectClusterListErrorMessage}}
+  <a *ngIf="isAdmin"
+  href="settings/seeds">check Seed Configration</a></div>
```

Comment:
```suggestion
  href="settings/seeds">Check Seed Configuration</a></div>
```

Reply:
Is this link highlighted (blue color and pointer cursor on hover) like other links in our application?

Reply:
yes i just update the pic in the description above